src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -1,6 +1,23 @@
 @import "scss/constants";
 
 .music-bar-container {
+  margin-top: 40px;
+
+  &--tableview {
+    margin-top: 0;
+
+    .music-bar-content {
```

Comment:
Basically my thinking here is that if can 'unnest' this, then it will probably be easier to put all of the mediaqueries 'together', which I think would make this feel a lot cleaner/simpler to follow.

Reply:
The reason it is nested is because it should only apply, when there is tableView status. I do agree it looks a bit cluttered, but ` it will probably be easier to put all of the mediaqueries 'together',` we don't really have much of them + this is considered to be a `temporary/quick` polish. I expect the whole component to be rewritten in the nearest future(Probably after the Auditorium)

Reply:
Fair enough. Not a blocker from me. Above was suggested in line with you 'not being particularly happy with how it was implemented', and wanting to offer suggestions/alternatives that feel cleaner.

Reply:
https://github.com/sparkletown/internal-sparkle-issues/issues/501